frame/nomination-pools/src/lib.rs

```diff
@@ -582,19 +582,200 @@ pub struct PoolRoles<AccountId> {
 	pub state_toggler: Option<AccountId>,
 }
 
+/// Pool commission.
+///
+/// The pool `root` can set commission configuration after pool creation. By default, all commission
+/// values are `None`. Pool `root`s can set `max` and `throttle` configurations before setting an
+/// initial commission `current` value - the commission percentage and payee of commission.
+///
+/// The `max` commission value can only be decreased after the initial value is
```

Comment:
line wrapping is too short. Can use this: https://marketplace.visualstudio.com/items?itemName=stkb.rewrap

Reply:
Nice tool, but applying it is now making everything a lot shorter, including the heading docs. It is formatting very differently to `cargo +nightly fmt`